tools/export/__init__.py

```diff
@@ -28,8 +28,8 @@
 from project_generator_definitions.definitions import ProGenDef
 
 EXPORTERS = {
-    'uvision': uvision4.Uvision4,
-    'uvision4': uvision4.Uvision4,
+    'uvision': uvision5.Uvision5,
```

Comment:
Did you make a new file or rename the uvision4 file? Because we refer to both as a module (uvision4. and uvision5.) here and the next line.